modules/configuring-block-volume-sharing-with-LUN-disks.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
+//
+// * * virt/virtual_machines/virtual_disks/virt-configuring-shared-volumes-for-vms.adoc
+
+:_content-type: PROCEDURE
+[id="configuring-block-volume-sharing-with-LUN-disks_{context}"]
+= Configuring block volume sharing with (LUN) disks
+
+You can configure a LUN-backed block mode virtual machine (VM) disk to be shared among multiple virtual machines by enabling SCSI perisistent reservation. Enabling this method allows you to use advanced SCSI commands, such as those required for a Windows Failover Clustering implmentation, against the underlying storage.
+
+A disk of type `LUN` exposes the volume as a LUN device to the VM. This allows the VM to execute arbitrary iSCSI command passthrough.
+
+.Prerequisites
+
+* A SCSI persistent reservation must be enabled by setting a feature gate.
```

Comment:
can we mention which feature gate?

Reply:
I am also nto sure if this can be enabled after the HyperConverged is created.

Reply:
@jobbler, this particular topic is no longer being used. (Mentioned in the Additional info section at the top of the PR.) But please look at the configuring-disk-sharing-by-using-LUN.adoc file and see if what I have there covers what should be said.

Reply:
the other file covers what I suggested. please forget about my previous comments.